src/client/pkg/tracing/tracing.go

```diff
@@ -0,0 +1,187 @@
+package tracing
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+	"sync"
+	"time"
+
+	otgrpc "github.com/opentracing-contrib/go-grpc"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/uber/jaeger-client-go"
+	jaegercfg "github.com/uber/jaeger-client-go/config"
+	"google.golang.org/grpc"
+)
+
+// JaegerServiceName is the service name used when the client reports traces
+// to Jaeger
+const JaegerServiceName = "pachd"
```

Comment:
Why is the Jaeger service named pachd and not jaeger?

Reply:
This is the name of the service pachd uses to identify itself when talking to Jaeger. I.e. in Jaeger you'll see traces for `pachd.PutFile` rather than `jaeger.PutFile`. I can rename the variable, I'm just not sure what would be a better name.

Reply:
Clarified the comment for JaegerServiceName